### PR TITLE
[#12048] Add is migrated flag to datastore account

### PR DESCRIPTION
**Pull request by daongochieu2810**
Part of #12048

Outline of Solution

1. Add isMigrated flag to entity/Account and entity/AccountAttributes

### CODE REVIEW

File: src/main/java/teammates/common/datatransfer/attributes/AccountAttributes.java [hunk 1]
```diff
@@ -22,6 +22,7 @@ public final class AccountAttributes extends EntityAttributes<Account> {
     private String email;
     private Map<String, Instant> readNotifications;
     private Instant createdAt;
+    private boolean isMigrated;
 
     private AccountAttributes(String googleId) {
         this.googleId = googleId;
```
Comment (zhaojj2209) [inline src/main/java/teammates/common/datatransfer/attributes/AccountAttributes.java:28]: Should we add `this.isMigrated = false` here?

Comment (daongochieu2810) [reply src/main/java/teammates/common/datatransfer/attributes/AccountAttributes.java:28]: I think the default value for `isMigrated` is false, so we might not need that

**Review by zhaojj2209 — APPROVED** [review]
LGTM

**Review by samuelfangjw — COMMENTED** [review]
Seems fine, but need to make sure it passes the "update only if change" check in AccountsDb (and that it can be updated). Could you help to check that the entity will update properly for accounts if only the isMigrated flag is changed (and course too)? Thanks!

**Review by samuelfangjw — APPROVED** [review]

**Pull request merged**